tutorials/quickstarts/GettingStartedWithPythonSDK.ipynb

```diff
@@ -0,0 +1,715 @@
+      "execution_count": null,
+      "metadata": {
+        "gather": {
+          "logged": 1612965865707
+        },
+        "jupyter": {
+          "outputs_hidden": false,
+          "source_hidden": false
+        },
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        }
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "Notice how the script gets data and saves models:\n",
+        "\n",
+        "+ The training script reads an argument to find the directory containing the data.  When you submit the job later, you point to the dataset for this argument:\n",
+        "`parser.add_argument('--data-folder', type=str, dest='data_folder', help='data directory mounting point')`\n",
+        "\n",
+        "\n",
+        "+ The training script saves your model into a directory named outputs. <br/>\n",
+        "`joblib.dump(value=clf, filename='outputs/sklearn_mnist_model.pkl')`<br/>\n",
+        "Anything written in this directory is automatically uploaded into your workspace. You'll access your model from this directory later in the tutorial.\n",
+        "\n",
+        "The file `utils.py` is referenced from the training script to load the dataset correctly.  This script is also copied into the script folder so that it can be accessed along with the training script."
+      ],
+      "metadata": {
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        }
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "### Configure the training job\n",
+        "\n",
+        "Create a [ScriptRunConfig]() object to specify the configuration details of your training job, including your training script, environment to use, and the compute target to run on. Configure the ScriptRunConfig by specifying:\n",
+        "\n",
+        "* The directory that contains your scripts. All the files in this directory are uploaded into the cluster nodes for execution. \n",
+        "* The compute target.  In this case you will point to local compute\n",
+        "* The training script name, train.py\n",
+        "* An environment that contains the libraries needed to run the script\n",
+        "* Arguments required from the training script. \n",
+        "\n"
+      ],
+      "metadata": {
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        }
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "An Environment defines Python packages, environment variables, and Docker settings that are used in machine learning experiments. Here you will be using a curated environment that has already been made available through the workspace. \n",
+        "\n",
+        "Read [this article](https://docs.microsoft.com/en-us/azure/machine-learning/how-to-use-environments) if you want to learn more about Environments and how to use them."
```

Comment:
1. no "/en-us" in links
2. generally avoid having links like this (they break often)

Reply:
Removed "/en-us/" from all links, thanks for pointing out! Should there be a general rule or something similar that says try to avoid having links in the notebooks that are likely to break?